src/reducers/headerSections.js

```diff
@@ -61,6 +61,14 @@ export default handleActions(
       });
       return newState;
     },
+    [getLatestBlockFulfilled]: (state, {payload}) => {
```

Comment:
So this is the code added here that searches through the array of sections, and then if the tag matches the last block tag then set the value to the last number. Is this the standard way to use actions in Redux? Feels like there's got to be a better way.

Reply:
I'm not that familiar with Redux but I can research